crosstests/google-java/src/test/kotlin/build/buf/connect/crosstest/CrossTest.kt

```diff
@@ -376,6 +377,138 @@ class CrossTest(
         assertThat(countDownLatch.count).isZero()
     }
 
+    @Test
+    fun emptyUnaryBlocking(): Unit = runBlocking {
+        val response = testServiceConnectClient.emptyCallBlocking(empty {}).execute()
+        response.failure {
+            fail<Unit>("expected error to be null")
+        }
+        response.success { success ->
+            assertThat(success.message).isEqualTo(empty {})
```

Comment:
Is there a guarantee that this is actually called? Looks like this test would pass if it was never invoked

Reply:
Same with other tests below

Reply:
I await for the countdown latch and verify that it got counted down (that the value is zero)

Reply:
For the blocking calls, it would call those as they're just for determining if the response was successful or not.